crates/runtime/src/pallets.rs

```diff
@@ -31,6 +31,14 @@ pub const SN_GOERLI_CHAIN_ID: Felt252Wrapper = Felt252Wrapper(starknet_ff::Field
     398700013197595345,
 ]));
 
+/// ChainId for Starknet Mainnet
```

Comment:
Now that we have multiple values. I would add this under a new primitive crate `mp-chain-id` with both values being hardcoded there.

And can then import it where ever it's used

Reply:
Done, tell me if any changes are needed. I did not called the constant in starknet-rpc-test since it does not match the crate three.